backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/material/cockpit/MaterialCockpitRowsLoader.java

```diff
@@ -100,15 +103,20 @@ public List<MaterialCockpitRow> getMaterialCockpitRows(
 		final List<I_QtyDemand_QtySupply_V> quantitiesRecords = QtyDemandSupplyFilters
 				.createQuantitiesQueryFor(filters)
 				.list();
+
+		final List<WarehouseId> warehouseIds = quantitiesRecords.stream()
+				.map(qtyRecord -> WarehouseId.ofRepoId(qtyRecord.getM_Warehouse_ID()))
```

Comment:
do we know that `M_Warehouse_ID` is never `null`? if yes, its AD_Column could be flagged as mandatory

Reply:
no longer needed in the end